crypto/fipsmodule/service_indicator/service_indicator.c

```diff
@@ -217,6 +217,32 @@ static int is_md_fips_approved_for_verifying(int md_type, int pkey_type) {
   }
 }
 
+// custom_meth_invoked checks whether custom crypto was invoked in the |meth|
+// or |eckey_method| fields for a given |RSA| or |EC_KEY| respectively. For
+// |RSA| keys, custom verify and sign functionality is supported. For |EC_KEY|
+// keys, only custom sign functionality is supported.
+// Returns one if no custom crypto was invoked and zero otherwise.
+static int no_custom_meth_invoked(const EVP_PKEY_CTX *ctx) {
```

Comment:
* This name is a little confusing -- It's stated in the negative, which is harder to reason about.
* The default for both [`RSA_METHOD`](https://github.com/aws/aws-lc/blob/f89c9bec9aea680839ae10aef2d9ac627a9c08c3/crypto/fipsmodule/rsa/rsa_impl.c#L1299) and [`EC_KEY_METHOD`](https://github.com/aws/aws-lc/blob/f89c9bec9aea680839ae10aef2d9ac627a9c08c3/crypto/fipsmodule/ec/ec_key.c#L572) is an all-zero struct.  To better future proof this, can we just compare it to this "default" value, and if there have been any changes then we assume this operation was not FIPS compliant?

Reply:
Changed the name. Second point discussed offline.